content/api/commands/attachfile.md

```diff
@@ -0,0 +1,213 @@
+---
+title: attachFile
+---
+
+Attaches files to HTML5 input element, or simulated dragging files onto the browser.
+
+## Syntax
+
+```javascript
+cy.attachFile(file)
+cy.attachFile(file, options)
+cy.attachFile([file], options)
+cy.attachFile([file], options)
+```
+
+### Usage
+
+**<Icon name="check-circle" color="green"></Icon> Correct Usage**
+
+```javascript
+  cy.get('input[type=file]').attachFile('file.json')
+  cy.get('input[type=file]').attachFile(['file.json', 'file2.json'])
+  cy.get('input[type=file]').attachFile({
+    contents: Buffer.from('foo'),
+    fileName: 'file.json',
+    lastModified: Date.now(),
+  })
+```
+
+**<Icon name="exclamation-triangle" color="red"></Icon> Incorrect Usage**
+
+```javascript
+cy.attachFile('file.json') // Errors, cannot be chained off 'cy'
+cy.get('input[type=file]').attachFile('file contents') // Will attempt to find a file called 'file contents' on disk, probably not what you intended
+```
+
+### Arguments
+
+**<Icon name="angle-right"></Icon> file** **_(String, Array, Object or Buffer)_**
+
+Either a single file, or an array of them. A file can be:
+- A path to a file within the project root (the directory that contains the default `cypress.json` configuration file). Eg: `'path/to/file.json'`
+- `@alias` - An alias of any type, previously stored by `.as()`. Eg: `'@alias'`
+- A `Buffer()` containing binary data, such as that returned by `cy.readFile('file.json', { encoding: null })`. Eg: `Buffer.from('foo')`
+- An object with a non-null `contents` property, specifying details about the file. Eg: `{contents: '@alias', fileName: 'file.json'}`
+
+If an object is provided, it can have the following properties.
+
+| Option         | Description                                                                                                                                                                                                              |
+| -------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `contents`     | The contents of the file. This can be a string shorthand as described above, a `Buffer()` containing binary data or a non-Buffer object, which will be converted into a string with `JSON.stringify()` and utf8 encoded. |
+| `fileName`     | The name of the file. If `contents` is a path on disk, this defaults to the actual filename. In any other case, this defaults to an empty string.                                                                        |
+| `lastModified` | The file's last modified timestamp. This defaults to `Date.now()`.                                                                                                                                                       |
```

Comment:
Might be good to clarify the time format expected (I guess saying Date.now() kind of does that). I can just see people passing in different datetime formats or something and not understanding why it's not working.

Reply:
Done. Since I keep having to look up exactly how to get the timestamp from a js Date, I'm just adding a MDN link straight to the docs.